src/routes/api.ts

```diff
@@ -57,13 +57,15 @@ router.get('/explore', (req, res) => {
       ? networks[space.network] + 1
       : 1;
 
-    space.strategies.forEach(strategy => {
-      strategies[strategy.name] = strategies[strategy.name]
-        ? strategies[strategy.name] + 1
+    const uniqueStrategies = new Set<string>(space.strategies.map((strategy) =>strategy.name));
+    uniqueStrategies.forEach(strategyName => {
+      strategies[strategyName] = strategies[strategyName]
+        ? strategies[strategyName] + 1
         : 1;
     });
 
-    Object.keys(space.plugins || {}).forEach(plugin => {
+    const uniquePlugins = new Set(Object.keys(space.plugins || {}));
+    uniquePlugins.forEach(plugin => {
```

Comment:
I think plugins should be unique per proposal anyway. So this change would not be necessary but doesn't do any harm also of course. Just came to my mind because I'm working on the plugin system and actually wondered for a moment... could there be more than one instance of a plugin in one space? No, not really. That would just mean the plugin is lacking some "have multiple of this or that" functionality.

Reply:
Thanks for feedback, just get rid of this part of change for plugins.